documentation/adr/0003-api-language.md

```diff
@@ -0,0 +1,99 @@
+# API Language
+
+- **Status:** Draft 
+- **Last Modified:** 2023-06-26
+- **Related Issue:** [#27](https://github.com/HHS/grants-api/issues/27) 
+- **Deciders:** Lucas Brown, Aaron Couch, Sammy Steiner, Daphne Gold
+
+## Context and Problem Statement
+
+This ADR is to decide what programming language the API server for beta.grants.gov will use. This API server will initially be responsible for responding to queries to search a replicated database, but will grow to encompass all the backend responsibilities of grants.gov
+
+In addition to the Nava and HHS teams ability to develop and maintain an API in the chosen language, it is also important that this language lends itself to open source engagement.
+
+## Decision Drivers <!-- RECOMMENDED -->
+
+- Nava and HHS familiarity with language and common libraries
+- Performance of language and associated frameworks at scale
+- Support of Open Source community
+- Cost of necessary licenses, support, etc.
+
+## Options Considered
+
+- Python
+- Javascript and Node
+- Java
+
+
+## Decision Outcome <!-- REQUIRED -->
+Pending Discussion and Decision
+
+<!-- Chosen option: "{option 1}", because {justification. e.g., only option which meets a key decision driver | which satisfies x condition | ... }. -->
+
+### Positive Consequences <!-- OPTIONAL -->
+Pending Discussion and Decision
+
+<!-- 
+- {e.g., improved performance on quality metric, new capability enabled, ...}
+- ... 
+-->
+
+### Negative Consequences <!-- OPTIONAL -->
+Pending Discussion and Decision
+
+<!-- 
+- {e.g., decreased performance on quality metric, risk, follow-up decisions required, ...}
+- ...
+-->
+
+
+## Pros and Cons of the Options <!-- OPTIONAL -->
+
+
+### Python
+
+Python as a scripting language is fast and productive. Web-based applications can be created quickly and the code is highly readable. Python syntax is easy to understand, well-defined and extensible. APIs built over python are highly scalable and have an unmatched speed.
+
+- **Pros**
+  - Nava and HHS experience with Python and APIs built with Python
```

Comment:
Let's add something about broad developer familiarity, outside of just HHS/Nava, because that determines whether it's easy to hire people, whether it's easy for open source contributors, etc. 

I would think Python and Node/JS have relatively equal popularity, with Java kind of falling behind, it doesn't seem as commonly used anymore by modern applications.